_data/authors.yml

```diff
@@ -55,3 +55,10 @@ ben_mansheim:
   github: //github.com/bmansheim
   twitter:
   bio: "Technical Writer"
+
+gary_niemen:
+  name: Gary Niemen
+  picture: https://docslikecode.com/images/gary-niemen.png
```

Comment:
I pulled this image from your GitHub avatar - it won't show in the preview but will appear once published to the live site.